### DISCUSSION TITLE
Ready to push?

### DISCUSSION THREAD

**Discussion by CarlosPietrasanta** [CarlosPietrasanta/Tafe230130#9] [Announcements]
Hey guys, I think we've reached at point were we are ready to start pushing and merging our features.

Who is ready to start?

**Comment by ozdotcom** [comment]
Carlos 
I am ready to push my changes - Mortgage Calculator


Sharryn

**Comment by ozdotcom** [comment]
Merge on the development branch running ok, you can continue with your feature branch.

Sharryn

**Comment by Jasonlovedataanalyst** [comment]
Carlos
I am ready to push my changes - Trip Calculator

Jason

**Comment by Jasonlovedataanalyst** [comment]
Merge on the development branch running fine, you can continue with your feature branch.

Jason

**Comment by CarlosPietrasanta** [comment]
Excellent, I pulled the latest version of development and I have your features running. I'm ready to push mine.

**Comment by CarlosPietrasanta** [comment]
All 3 features merged and development branch was updated.

I will let you know once I completed all the required steps on Section D, so you can continue working on Section E and F.

Have a good night.
